mapbox_tilesets/errors.py

```diff
@@ -24,4 +24,8 @@ class TilesetNameError(TilesetsError):
     """
 
     def __init__(self, tileset_id):
-        self.message = f"{tileset_id} is not a valid Tileset ID"
+        self.tileset_id = tileset_id
+        self.message = "Invalid Tileset ID"
+
+    def __str__(self):
+        return f"{self.tileset_id} -> {self.message}"
```

Comment:
f strings are python 3.6+, so we should use:
```suggestion
        return "{tileset_id} -> {self.message}".format(tileset_id=self.tileset_id)
```

Reply:
👍 done in 9f8e5cc2f9bb29003edd64ed2308b1452e7462f1